pep-0544.txt

```diff
@@ -1232,11 +1232,11 @@ Another potentially problematic case is assignment of attributes
 *after* instantiation::
 
   class P(Protocol):
-      x: int
+      y: int
```

Comment:
This change seems gratuitous. The previous `x` is in a different example.

Reply:
I was specifically referring to confusion with the use of `x` later in the code block, but I did think this change would be deemed unnecessary.